docs/proposals/ListView.md

```diff
@@ -0,0 +1,153 @@
+# Component Name
+
+The `ListPage` component will provide the structure and parts needed to display
+a list view.
+
+## Design Patterns
+
+There is a desire to provide developers with page templates to both speed up the
+development of pages but also provide our users with a more consistent
+experience.
+
+A first step in this is imagining what a common `ListPage` component would look
+like. It would need to have some level of flexibility to allow for use on
+different pages but still provide common "slots" or areas where.
+
```

Comment:
Is it worth mentioning here how this pattern should be addressed over time? I think we're starting simple here, which is good. 
But a proposal for a general layout template approach like this should probably be discussed or considered by a number of people... how do we work with designers and developers to get their input on this?